hspec-junit-formatter.cabal

```diff
@@ -41,14 +40,28 @@ library
   build-depends:
       base <5
     , conduit
+    , containers
     , directory
     , exceptions
-    , hashable
-    , hspec
-    , hspec-core
-    , resourcet
-    , temporary
+    , filepath
+    , hspec-core <=2.8.1
```

Comment:
We aren't doing upper bounds anymore. And don't you want a _lower_ bound now?

Reply:
Lol, this was suppose to be a lower. It'll take some squinting for me to figure out if it is an upper or lower :laughing: